excellent/functions/builtin.go

```diff
@@ -1512,6 +1513,30 @@ func TimeFromParts(env envs.Environment, hour, minute, second int) types.XValue
 // Array Functions
 //------------------------------------------------------------------------------------------
 
+// Contains returns whether `array` contains `value`.
+//
+//	@(contains(array("a", "b", "c"), "a")) -> true
+//	@(contains(array(1, 2, 3), 4)) -> false
+//
+// @function contains(array, value)
+func Contains(env envs.Environment, arg1 types.XValue, value types.XValue) types.XValue {
+	array, xerr := types.ToXArray(env, arg1)
+	if xerr != nil {
+		return xerr
+	}
+	if types.IsXError(value) {
+		return value
+	}
+
+	for i := 0; i < array.Count(); i++ {
+		if types.Equals(array.Get(i), value) {
```

Comment:
note that this isn't using "textual equality" that we use for `x = y`.. but it's consistent with how other array functions work like `unique`. I guess we could add "true equality" as `x == y` and then say that functions like this use that.